modules/cli/src/main/scala/coursier/cli/bootstrap/BootstrapSpecificOptions.scala

```diff
@@ -2,63 +2,82 @@ package coursier.cli.bootstrap
 
 import caseapp.{ExtraName => Short, HelpMessage => Help, ValueDescription => Value, _}
 import coursier.install.RawAppDescriptor
+import coursier.cli.options.OptionGroup
 
 // format: off
 final case class BootstrapSpecificOptions(
+  @Group(OptionGroup.bootstrap)
   @Short("o")
     output: Option[String] = None,
+  @Group(OptionGroup.bootstrap)
+  @Hidden
```

Comment:
This one shouldn't be hidden IMO. It's even suggested when the file already exists.

Reply:
Done